test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -69,15 +67,12 @@ func makeCudaAdditionDevicePluginTestPod() *v1.Pod {
 	return testPod
 }
 
-func isClusterRunningCOS(f *framework.Framework) bool {
+func logOSImages(f *framework.Framework) {
```

Comment:
Do we also want to remove COS from testNvidiaGPUsOnCOS() function name and the test func description in line 177?

Reply:
Updated.

Reply:
Maybe also remove "on Container Optimized OS only" on line 177?

Reply:
Done.